### PR TITLE
[Enterprise-4.8]: Corrected two typos in the installation documentation.

### PR DESCRIPTION
**Pull request by EricPonvelle**
This PR corrects two typos that caused confusion. 

Version: 4.8+

Preview:

1. https://deploy-preview-34461--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.html#nw-gcp-global-access-configuration_installing-restricted-networks-gcp-installer-provisioned
2. https://deploy-preview-34461--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_openstack/installing-openstack-user.html#installation-osp-provider-network-preparation_installing-openstack-user

### CODE REVIEW

File: modules/installation-osp-provider-network-preparation.adoc [hunk 1]
```diff
@@ -46,7 +46,7 @@ Provider networks must be owned by the {rh-openstack} project that is used to cr
 
 * Verify that the provider network can reach the {rh-openstack} metadata service IP address, which is `169.254.169.254` by default.
 +
-Depending on your {rh-openstack} SDN and networking service configuration, you might need to create provide the route when you create the subnet. For example:
+Depending on your {rh-openstack} SDN and networking service configuration, you might need to provide the route when you create the subnet. For example:
```
Comment (maxwelldb) [inline modules/installation-osp-provider-network-preparation.adoc:49]: "when" or "after?"

File: modules/nw-gcp-installing-global-access-configuration.adoc [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 [id="nw-gcp-global-access-configuration_{context}"]
 = Create an Ingress Controller with global access on GCP
-You can create an Ingress Controller with global access on a new GCP cluster when your. Global access is only available to Ingress Controllers using internal load balancers.
+You can create an Ingress Controller that has global access to the GCP cluster. Global access is only available to Ingress Controllers using internal load balancers.
```
Comment (maxwelldb) [inline modules/nw-gcp-installing-global-access-configuration.adoc:8]: "to the" or "to a"?

Comment (maxwelldb) [inline modules/nw-gcp-installing-global-access-configuration.adoc:8]: Do we define GCP on first use, or is it exempt from that rule?

Comment (maxwelldb) [reply modules/nw-gcp-installing-global-access-configuration.adoc:8]: Also, "using internal," "by using internal," or "that use internal?"

**Review by maxwelldb — APPROVED** [review]
lgtm. Asked a few clarifying questions.

**Comment by skrthomas** [comment]
LGTM!

**Comment by maxwelldb** [comment]
Merging, as Vikram says that this is fine.

**Comment by maxwelldb** [comment]
@EricPonvelle Could you take a look at #34536?

File: modules/nw-gcp-installing-global-access-configuration.adoc (hunk 2 above)
Comment (EricPonvelle) [reply modules/nw-gcp-installing-global-access-configuration.adoc:8]: @yunjiang29 Can you tell me which of these options it would be?

Comment (yunjiang29) [reply modules/nw-gcp-installing-global-access-configuration.adoc:8]: @tsze-redhat PTAL, thanks

**Comment by tsze-redhat** [comment]
@lihongan 
Looks like the paragraph containing the typos here originated from
https://issues.redhat.com/browse/OSDOCS-1834

Could you PTAL at this PR (and the paragraph if you haven't already done so)?

FWIW, I would like to have "Optional" at beginning of the section if it is truly optional to be in line with other optional steps.

Thanks.

**Comment by lihongan** [comment]
@tsze-redhat There are two modified doc in this PR but only the second one `modules/nw-gcp-installing-global-access-configuration.adoc` is related to ingress and the part looks good to me.

And I just saw `Optional` for some steps but not for sections, so I don't think it is required here.

**Comment by tsze-redhat** [comment]
Thanks @lihongan .

**Pull request merged**